Makefile

```diff
@@ -21,12 +21,32 @@ help: ## Prints all the targets in all the Makefiles
 path_build: ## build the path binary
 	go build -o bin/path ./cmd
 
+.PHONY: path_run
+path_run: path_build
+	@if [ ! -f ./bin/config/.config.yaml ]; then \
+		echo "#########################################################################################"; \
+		echo "### ./bin/config/.config.yaml does not exist, use ONE the following to initialize it: ###"; \
+		echo "A. make copy_shannon_config                                                           ###"; \
+		echo "B. make copy_morse_config                                                             ###"; \
+		echo "#########################################################################################"; \
+		exit; \
+	fi; \
+	(cd bin; ./path)
+
+.PHONY: path_up
+path_up: config_shannon_localnet ## Run the PATH gateway and all related dependencies
```

Comment:
Shouldn't this also include the targets for `make localnet_up`, eg.

```make
.PHONY: localnet_up
localnet_up: dev_up config_path_secrets ## Brings up local Tilt development environment (using kind cluster)
	@tilt up

.PHONY: localnet_down
localnet_down: dev_down ## Tears down local Tilt development environment (using kind cluster)
```
https://github.com/buildwithgrove/path/blob/b7a8774704dad447083e3a1017b4a9b219167f1f/Makefile#L206C1-L211C94